### DISCUSSION TITLE
Why livewire session is too slow?

### DISCUSSION THREAD

**Discussion by fadilbjm** [livewire/livewire#3872] [1. Help]
Hello guys! i've been release my project in production. But when i testing it , livewire is too slow (need 5s to finish load). I think it's because my query so i install laravel debugbar, after that i look my query is normal but the session query is too slow and need to take more than 3s
![image](https://user-images.githubusercontent.com/17346613/134811700-a87f4fd4-bf47-4acc-adff-4b1235f115ee.png)

and here in my computer working fine
![image](https://user-images.githubusercontent.com/17346613/134811765-be5cb815-33b1-4a59-9580-9dbf1d36db0f.png)

**Comment by ju5t** [comment]
Are you sure this isn't a database issue?

**Comment by ariaieboy** [comment]
> Hello guys! i've been release my project in production. But when i testing it , livewire is too slow (need 5s to finish load). I think it's because my query so i install laravel debugbar, after that i look my query is normal but the session query is too slow and need to take more than 3s
> ![image](https://user-images.githubusercontent.com/17346613/134811700-a87f4fd4-bf47-4acc-adff-4b1235f115ee.png)
> 
> and here in my computer working fine
> ![image](https://user-images.githubusercontent.com/17346613/134811765-be5cb815-33b1-4a59-9580-9dbf1d36db0f.png)

its not about livewire
if you look at the query sourcec its come from the laravel it self not livewire
it might be an index problem in your production database that it tooks that much time to return the value you want.

**Comment by fadilbjm** [comment]
So i need to index the sessions table to solving this issue?

Pada tanggal Sen, 27 Sep 2021 06.46, AriaieBOY ***@***.***>
menulis:

**Reply by ariaieboy** [reply]
if you post your session table schema i could help you better 
it also can be a problem with your database configuration

**Reply by fadilbjm** [reply]
![image](https://user-images.githubusercontent.com/17346613/134915522-87bb7070-5f0b-48e1-b423-02cfdacb047a.png)
here's default jetstream migration

**Reply by ariaieboy** [reply]
> ![image](https://user-images.githubusercontent.com/17346613/134915522-87bb7070-5f0b-48e1-b423-02cfdacb047a.png)
> here's default jetstream migration

as i said it's can be from your db configuration some versions of mysql have limitation on indexing the string columns you must check the mysql version and it's configuration in your production server
